packages/mpg123/mpg123.0.3/opam

```diff
@@ -0,0 +1,41 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "MP3 decoding library"
+description: "OCaml bindings to libmpg123"
+maintainer: ["Michael Bacarella <m@bacarella.com>"]
+authors: ["Michael Bacarella <m@bacarella.com>"]
+license: "LGPL-2.1-or-later with OCaml-LGPL-linking-exception"
+homepage: "https://github.com/mbacarella/mpg123"
+bug-reports: "https://github.com/mbacarella/mpg123/issues"
+depends: [
+  "ocaml" {>= "4.03"}
+  "dune" {>= "2.7" & >= "2.0"}
```

Comment:
```suggestion
  "dune" {>= "2.7"}
```
You can remove `dune` among the dependencies in the dune-project file, it is added automatically (and will get rid of this lint change)